apps/remix-ide/src/walkthroughService.js

```diff
@@ -28,6 +28,59 @@ export class WalkthroughService extends Plugin {
     })*/
   }
 
+  startRecorderW () {
+    introJs().setOptions({
+      steps: [{
+        title: 'Transactions Recorder',
+        intro: 'Save transactions (deployed contracts and function executions) and replay them in another environment e.g Transactions created in Remix VM can be replayed in the Injected Provider.Click to launch the Home tab that contains links, tips, and shortcuts..',
+        element: document.querySelector('#udappRecorderCard'),
+        tooltipClass: 'bg-light text-dark',
+        position: 'right',
+        highlightClass: 'bg-light border border-warning'
+      },
+      {
+        element: document.querySelector('#udappRecorderUseLatest'),
+        title: 'Transactions Recorder',
+        intro: 'If set the recorder will run transactions using the latest compilation result.',
+        tooltipClass: 'bg-light text-dark',
+        position: 'right',
+        highlightClass: 'bg-light border border-warning'
+      },
+      {
+        element: document.querySelector('#udappRecorderSave'),
+        title: 'Transactions Recorder',
+        intro: 'Once there is a transaction, click the button Save to save the scenario.',
+        tooltipClass: 'bg-light text-dark',
+        position: 'right',
+        highlightClass: 'bg-light border border-warning'
+      },
+      {
+        element: document.querySelector('#udappRecorderRun'),
+        title: 'Transactions Recorder',
+        intro: 'Run previously saved scenario over any environment.',
```

Comment:
`Open a scenario file and click this button to run it against the current selected provider.`